### DISCUSSION TITLE
More examples

### DISCUSSION THREAD

**Discussion by LintaoAmons** [tamago324/nlsp-settings.nvim#29] [General]
~~~~~~~
More examples~~~~~~
~~~~~~~

**Comment by tamago324** [comment]
@LintaoAmons 
Thanks for your comment.
What do you want an example for?

**Reply by LintaoAmons** [reply]
In `examples` dir, I saw the `sumneko_lua.json`.
Maybe there could be more sane `json` files~~

`tsserver`... etc.
